changelog

```diff
@@ -1,3 +1,135 @@
+Version 0.10.5
+  * Big changes:
+    * Mining Stuff (new attributes, new outfits, new ships, new mechanics, etc... @Amazinite will expand upon this).
+  * Bug fixes:
+    * Content bugs:
+      * Typo fixes. (@Corraban2, @opusforlife2, @Wedge009, @jarmokivekas, @Anarchist2, @Tangle10, @Saugia, @tibetiroka, @bene-dictator, @ziproot, @mOctave, @alexrovw, (@The-Legendary-M),
+      * Fixed the location filters for some pirate escort jobs. (@ziproot)
+      * Added a missing "goto" in "Southern Finance 6". (@bene-dictator)
+      * Added a missing departure distance to Dokdobaru. (@Arachi-Lover)
+      * Various fixes to "Navy Resupply [0]". (@LazerLit)
+      * Added missing conditions in the Heliarch intro. (@ziproot)
+      * Added some missing system attributes. (@Tangle10)
+      * Added a missing `goto` in "Saryd University Lecture". (@RedDeadUndead)
+      * Fixed/adjusted a few things in the "Early South" mission set. (@bene-dictator)
+      * Corrected several outdated conditions in "FW Bloodsea 1.1". (@RisingLeaf)
```

Comment:
```suggestion
      * Corrected several outdated conditions in "FW Bloodsea 1.1". (@bene-dictator)
```
Is this some way of Rising telling me to finish Dead Man's Path?

Reply:
Just realised that #9636 caused the bugs that I fixed in #9647. Heh heh.

~~Just credit both of us because otherwise the original bug would have remained.~~
```suggestion
      * Corrected several outdated conditions in "FW Bloodsea 1.1". (@RisingLeaf, @bene-dictator)
```